### ISSUE TITLE
Application error a client side exception has occured

### ISSUE THREAD

**Issue by Rowan-Paul** [Rowan-Paul/project-argus#26]
Some pages give random client side exceptions
```
Application error: a client-side exception has occurred (see the browser console for more information)
```

https://www.projectarg.us/shows/rome-2005/seasons/2

**Comment by Rowan-Paul** [comment]
Probably fixed by #24

**Comment by Rowan-Paul** [comment]
I have not come across it since the #24 so I'm closing this
